internal_transfers/actions/src/database.ts

```diff
@@ -5,14 +5,22 @@ import tables from "@databases/pg-typed";
 import ConnectionPool from "@databases/pg/lib/types/Queryable";
 import DatabaseSchema from "./__generated__";
 import { EventMeta, SettlementEvent, TokenImbalance } from "./models";
-import { SettlementSimulationData } from "./accounting";
+import { MinimalTxData, SettlementSimulationData } from "./accounting";
 import { SimulationData } from "./simulate/interface";
 
 export { sql };
-const { settlements, internalized_imbalances, settlement_simulations } =
-  tables<DatabaseSchema>({
-    databaseSchema: require("./__generated__/schema.json"),
-  });
+const {
+  settlements,
+  internalized_imbalances,
+  settlement_simulations,
+  tx_receipts,
+} = tables<DatabaseSchema>({
+  databaseSchema: require("./__generated__/schema.json"),
+});
+
+function pgHash(txHash: string) {
```

Comment:
Nit: Give this function a more meaningful name that describes what it does.

Reply:
any suggestions? I was under the impression that it did describe what it does (`postgresHash`)... maybe `pgHexBytes`?

Reply:
Issue: https://github.com/cowprotocol/solver-rewards/issues/273